code/_onclick/click.dm

```diff
@@ -117,6 +117,10 @@
 		var/obj/mecha/M = loc
 		return M.click_action(A,src,params)
 
+	if(iscustmech(loc))
+		var/mob/living/exosuit/M = loc
```

Comment:
don't use one character vars, for your sake, do something like "var/mob/living/exosuit/our_mech"

Reply:
So while I totally agree with this.


In this particular case we are not able to do so without refactoring the code.

Because these standards were set by Shiptest and each edit takes us farther from shiptest code... and makes it harder to pull new PR stuff from shiptest.


In our own code... and files which will not be touched/depend on shiptest... then yes... 100% better variables.

Reply:
> So while I totally agree with this.
> 
> In this particular case we are not able to do so without refactoring the code.
> 
> Because these standards were set by Shiptest and each edit takes us farther from shiptest code... and makes it harder to pull new PR stuff from shiptest.
> 
> In our own code... and files which will not be touched/depend on shiptest... then yes... 100% better variables.

having more than one character shouldn't matter too much for modularity